### PR TITLE
Populated pre-CNA JSON for published advisories.

### PR DESCRIPTION
**Pull request by ViperGeek**
CVE-2015-3006	https://kb.juniper.net/InfoCenter/index?page=content&id=JSA10678
CVE-2015-5361	http://kb.juniper.net/InfoCenter/index?page=content&id=JSA10706

### CODE REVIEW

File: 2015/3xxx/CVE-2015-3006.json [hunk 1]
```diff
@@ -1,8 +1,50 @@
 {
```
Comment (cve-team) [inline 2015/3xxx/CVE-2015-3006.json]: CNA(s) of user (juniper) do not match owning CNA.

File: 2015/5xxx/CVE-2015-5361.json [hunk 2]
```diff
@@ -1,8 +1,54 @@
 {
```
Comment (cve-team) [inline 2015/5xxx/CVE-2015-5361.json]: CNA(s) of user (juniper) do not match owning CNA.

File: 2015/3xxx/CVE-2015-3006.json (hunk 1 above)
Comment (ViperGeek) [reply 2015/3xxx/CVE-2015-3006.json]: While I recently updated my dave@juniper.net PGP key from 0x879F65CDAB6E02A5 to 0xC5BBC32B0B71727A, I now see that I mistakenly used my personal PGP Key ID 0x5E7DB5AB7634153C for these commits.

Please let me know how to manually validate these commits.

File: 2015/5xxx/CVE-2015-5361.json (hunk 2 above)
Comment (ViperGeek) [reply 2015/5xxx/CVE-2015-5361.json]: While I recently updated my dave@juniper.net PGP key from 0x879F65CDAB6E02A5 to 0xC5BBC32B0B71727A, I now see that I mistakenly used my personal PGP Key ID 0x5E7DB5AB7634153C for these commits.

Please let me know how to manually validate these commits.

**Comment by ccoffin** [comment]
ViperGeek,

Because these are older CVE IDs that are owned by the MITRE CNA, we had to manually add them to our CPS. You should see your submission on the CVE web site and in the GitHub cvelist in the next couple of hours.

Regards,

Chris

**Pull request closed without merging**